server2/Arcadia.Assistant/Arcadia.Assistant.PushNotifications/PushNotifications.cs

```diff
@@ -0,0 +1,137 @@
+namespace Arcadia.Assistant.PushNotifications
+{
+    using System.Collections.Generic;
+    using System.Fabric;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Castle.Core.Internal;
+
+    using Contracts.Models;
+
+    using Interfaces;
+
+    using Microsoft.Extensions.Logging;
+    using Microsoft.ServiceFabric.Services.Communication.Runtime;
+    using Microsoft.ServiceFabric.Services.Runtime;
+
+    using Models;
+
+    /// <summary>
+    ///     An instance of this class is created for each service instance by the Service Fabric runtime.
+    /// </summary>
+    public sealed class PushNotifications : StatelessService
```

Comment:
don't you need to implement an interface?

Reply:
Fixed.